price-feeder/oracle/provider/kraken.go

```diff
@@ -259,18 +259,12 @@ func (p *KrakenProvider) handleWebSocketMsgs(ctx context.Context) {
 		case <-time.After(defaultReadNewWSMessage):
 			messageType, bz, err := p.wsClient.ReadMessage()
 			if err != nil {
-				if websocket.IsCloseError(err, websocket.CloseAbnormalClosure) {
-					p.logger.Err(err).Msg("WebSocket closed unexpectedly")
-					p.keepReconnecting()
+				if err != nil {
```

Comment:
@adamewozniak  I matched this to how mexc is handling errored messages. Is this correct or should it be left the same with "keepReconnecting" being replaced with reconnect.